test/test_spectral_ops.py

```diff
@@ -157,11 +158,13 @@ def test_fft_round_trip(self, device, dtype):
     def test_empty_fft(self, device, dtype):
         t = torch.empty(0, device=device, dtype=dtype)
         match = r"Invalid number of data points \([-\d]*\) specified"
-        fft_functions = [torch.fft.fft, torch.fft.ifft, torch.fft.hfft,
-                         torch.fft.irfft]
+        fft_functions = [torch.fft.fft, torch.fft.fftn,
```

Comment:
Not in this PR, but in a follow-up it'd be nice to see the fft functions added to the OpInfo database in common_methods_invocations.py. They could have their own base class derived from OpInfo. Registering them as OpInfos would make these tests a little nicer, and we'll be moving all `method_tests` registrations to OpInfos in the future, anyway.